compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -91,13 +97,24 @@ class ReplDriver(settings: Array[String],
   }
 
   private var rootCtx: Context = _
+  private var shouldStart: Boolean = _
   private var compiler: ReplCompiler = _
   private var rendering: Rendering = _
 
   // initialize the REPL session as part of the constructor so that once `run`
   // is called, we're in business
   resetToInitial()
 
+  override protected def command: CompilerCommand = ReplCommand
+
+  /** Try to run REPL if there is nothing that prevents us doing so.
+   *
+   * Possible reason for unsuccessful run are raised flags in CLI like --help or --version
+   */
+  final def tryRunning = if shouldStart then
+    println("Starting scala3 REPL...")
```

Comment:
I see this was here before, but I would just delete that line, the repl already prints enough stuff when it starts.

Reply:
Can remove